### PR TITLE
Add integration latestversion

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionAddress.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        private DateTimeOffset VersionTimestampAsDateTimeOffset { get; set; }
+
+        public Instant VersionTimestamp
+        {
+            get => Instant.FromDateTimeOffset(VersionTimestampAsDateTimeOffset);
+            set
+            {
+                VersionTimestampAsDateTimeOffset = value.ToDateTimeOffset();
+                VersionAsString = new Rfc3339SerializableDateTimeOffset(value.ToBelgianDateTimeOffset()).ToString();
+            }
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionAddress.cs:32]: No need to save version here.

File: src/ParcelRegistry.Projector/Infrastructure/Modules/ApiModule.cs [hunk 2]
```diff
@@ -150,8 +151,13 @@ private void RegisterIntegrationProjections(ContainerBuilder builder)
                 .RegisterProjectionMigrator<IntegrationContextMigrationFactory>(
                     _configuration,
                     _loggerFactory)
-                .RegisterProjections<ParcelLatestItemProjections, IntegrationContext>(
-                    context => new ParcelLatestItemProjections(context.Resolve<IOptions<IntegrationOptions>>()),
+                // .RegisterProjections<ParcelLatestItemProjections, IntegrationContext>(
+                //     context => new ParcelLatestItemProjections(context.Resolve<IOptions<IntegrationOptions>>()),
+                //     ConnectedProjectionSettings.Default)
```
Comment (ArneD) [inline src/ParcelRegistry.Projector/Infrastructure/Modules/ApiModule.cs:156]: uncomment?

File: src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                {
+                    var newParcelAddressVersion = parcelVersionAddress.CloneAndApplyEventInfo(
+                        message.Position,
+                        provenance.Timestamp);
+
+                    await context
+                        .ParcelVersionAddresses
+                        .AddAsync(newParcelAddressVersion, ct);
+                }
+            }
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs:55]: Every event should also have corresponding address(es) even if it doesn't change. Except when there are no address attached.

File: src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            long position,
+            CancellationToken ct)
+        {
+            var localAddresses = context
+                .ParcelVersionAddresses
+                .Local
+                .Where(x => x.ParcelId == parcelId && x.Position == position)
+                .ToList();
+
+            if (!localAddresses.Any())
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs:87]: I think you need to union, one can be local other can be in database?

File: src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs (hunk 3 above)
Comment (emalfroy) [reply src/ParcelRegistry.Projections.Integration/ParcelVersion/ParcelVersionExtensions.cs:55]: this code block will always execute, unless with removals of every address, then we don't add new versions of those addresses
`bool renewAddresses = true`

**Review by ArneD — APPROVED** [review]

**Pull request merged**